hawkular-alerts-actions-plugins/hawkular-alerts-actions-email/src/main/java/org/hawkular/alerts/actions/email/EmailPlugin.java

```diff
@@ -223,18 +235,46 @@ private void initMailSession() {
         return defaultProperties;
     }
 
-    @Override
-    public void process(PluginMessage msg) throws Exception {
-        Message message = createMimeMessage(msg);
-        Transport.send(message);
+    private void sendResult(Action action) {
```

Comment:
Is the email plugin the only plugin currently persisting results?  Or did I miss something in the other plugins?